roles/default/kiali-deploy/defaults/main.yml

```
@@ -207,6 +207,19 @@ kiali_defaults:
   kiali_feature_flags:
     istio_injection_action: true
     ui_defaults:
+      graph:
+        find_options:
+          - description: "Find: slow edges (> 1s)"
+            expression: "rt > 1000"
+          - description: "Find: unhealthy nodes"
+            expression:  "! healthy"
+          - description: "Find: unknown nodes"
+            Expression:  "name = unknown"
+        hide_options:
+          - description: "Hide: healthy nodes"
+            expression: "healthy"
+          - description: "Hide: unknown nodes"
+            expression:  "name = unknown"
```

Follow the conventions everywhere else in the operator - we do not indent lists twice. So this should be:
```suggestion
        find_options:
        - description: "Find: slow edges (> 1s)"
          expression: "rt > 1000"
        - description: "Find: unhealthy nodes"
          expression:  "! healthy"
        - description: "Find: unknown nodes"
          Expression:  "name = unknown"
        hide_options:
        - description: "Hide: healthy nodes"
          expression: "healthy"
        - description: "Hide: unknown nodes"
          expression:  "name = unknown"
```